pkg/plugins/platform/local_test.go

```diff
@@ -219,13 +223,11 @@ func TestLocalImportDB(t *testing.T) {
 			err = app.ImportDB(cachedArchive, "")
 			assert.NoError(err)
 
-			stdout := testcommon.CaptureStdOut()
-			err = app.Exec("db", true, "mysql", "-e", "SHOW TABLES;")
+			out, _, err := app.Exec("db", "mysql", "-e", "SHOW TABLES;")
```

Comment:
is the 2nd returned var actually used anywhere we're calling app.Exec? (stderr output i think) I don't think I've seen it in use.

Reply:
Yes, the tests now use it instead of trying to capture output in the fancy output-capture way.